mmv1/products/redis/api.yaml

```diff
@@ -285,3 +285,14 @@ objects:
         default_value: :READ_REPLICAS_DISABLED
         min_version: beta
         input: true
+      - !ruby/object:Api::Type::String
+        name: secondaryIpRange
+        description: |
+          Optional. Additional ip ranges for node placement, beyond those specified
```

Comment:
These docs don't quite match what I see on https://cloud.google.com/memorystore/docs/redis/reference/rest/v1beta1/projects.locations.instances - just wanted to confirm that this is intentional? Generally we try to only deviate from c.g.c in terms of field names or if there's Terraform-specific information

Reply:
fixed in https://github.com/GoogleCloudPlatform/magic-modules/pull/5793